pkg/api/api.go

```diff
@@ -180,6 +189,18 @@ func requestEncrypt(r *http.Request) bool {
 	return strings.ToLower(r.Header.Get(SwarmEncryptHeader)) == "true"
 }
 
+func requestPostageBatchId(r *http.Request) ([]byte, error) {
+	if h := strings.ToLower(r.Header.Get(SwarmPostageBatchIdHeader)); h != "" {
+		if len(h) != 64 {
+			return nil, errInvalidPostageBatch
+		}
+		return hex.DecodeString(h)
+	}
+	b := make([]byte, len(fallbackPostageBatch))
+	_ = copy(b, fallbackPostageBatch)
```

Comment:
I suppose that this approach is to have a fallback postage back clearly defined on the package level, instead here just to return `make([]byte, 32)`. Could you add a comment?